test/tests/security/pbs_job_security.py

```diff
@@ -0,0 +1,83 @@
+# applicable end user license agreement between Altair and Licensee.
+# In the absence of such agreement, the Altair standard end user
+# license agreement terms shall govern.
+
+from tests.security import *
+
+
+class Test_job_security(TestSecurity):
+    """
+    This test suite is for testing job security
+    """
+
+    def setUp(self):
+        TestSecurity.setUp(self)
+        a = {'job_history_enable': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+    def test_interactive_job_env_V(self):
+        """
+        This test checks if the PBS job related cookies are passed on to
+        the job that is submitted within the interactive job with -V.
+        """
+        self.pbs_conf = self.du.parse_pbs_config(self.server.shortname)
+        self.exec_path = os.path.join(self.pbs_conf['PBS_EXEC'], "bin")
+        j = Job(TEST_USER)
+        set_attr = {ATTR_inter: ''}
+        j.set_attributes(set_attr)
+        j.interactive_script = [('export PATH=$PATH:%s' %
+                                 self.exec_path, '.*'),
+                                ('qsub -V -- /bin/sleep 5', '.*'),
+                                ('sleep 30', '.*')]
+        jid = self.server.submit(j)
+        jid2 = str(int(jid.split('.')[0]) + 1)
+        self.server.expect(JOB, {'job_state=R': 2}, count=True)
+        stat = self.server.status(JOB, id=jid2, extend='x')
+        for s in stat:
+            variable_list = s.get('Variable_List', '')
+            if 'PBS_JOBCOOKIE' in variable_list or \
+                    'PBS_INTERACTIVE_COOKIE' in variable_list:
+                self.fail("Job Cookie(s) passed on to the inner job.")
+
+    def test_interactive_job_env_v(self):
+        """
+        This test checks if the PBS job related cookies are passed on to
+        the job that is submitted within the interactive job
+        with -v PBS_JOBCOOKIE.
+        """
+        self.pbs_conf = self.du.parse_pbs_config(self.server.shortname)
+        self.exec_path = os.path.join(self.pbs_conf['PBS_EXEC'], "bin")
+        j = Job(TEST_USER)
+        set_attr = {ATTR_inter: ''}
+        j.set_attributes(set_attr)
+        j.interactive_script = [('export PATH=$PATH:%s' %
+                                 self.exec_path, '.*'),
+                                ('qsub -v PBS_JOBCOOKIE -- \
+                                    /bin/sleep 5', '.*'),
+                                ('sleep 30', '.*')]
+        jid = self.server.submit(j)
+        jid2 = str(int(jid.split('.')[0]) + 1)
+        self.server.expect(JOB, {'job_state=R': 2}, count=True)
+        stat = self.server.status(JOB, id=jid2, extend='x')
+        for s in stat:
+            variable_list = s.get('Variable_List', '')
+            if 'PBS_JOBCOOKIE' in variable_list or \
+                    'PBS_INTERACTIVE_COOKIE' in variable_list:
+                self.fail("Job Cookie(s) passed on to the inner job.")
+
+    def tearDown(self):
+        TestSecurity.tearDown(self)
```

Comment:
You shouldn't need this.  You are overriding TestSecurity's tearDown() just to call it.